### DISCUSSION TITLE
询问

### DISCUSSION THREAD

**Discussion by qj65mmm** [ThisIs-Developer/Llama-2-GGML-Medical-Chatbot#47] [General]
你好，我遇到了这个错误：The de-serialization relies loading a pickle file. Pickle files can be modified to deliver a malicious payload that results in execution of arbitrary code on your machine.You will need to set allow_dangerous_deserialization
to True
to enable deserialization. If you do this, make sure that you trust the source of the data. For example, if you are loading a file that you created, and know that no one else has modified the file, then this is safe to do. Do not set this to True
if you are loading a file from an untrusted source (e.g., some random site on the internet.).   应如何解决呢？请教一下，谢谢！

**Comment by ThisIs-Developer** [comment]
如果担心安全问题，可以尝试用 JSON 或 Joblib 替代 Pickle。

**Python Pickle Risks and Safer Serialization Alternatives:** https://www.arjancodes.com/blog/python-pickle-module-security-risks-and-safer-alternatives/
